README.md

```diff
@@ -1,2 +1,67 @@
-# fylo-project-review
-This project is made for learning purposes in order to know about the concept of code reviewer.
+# Frontend Mentor - Fylo data storage component solution
+
+This is a solution to the [Fylo data storage component challenge on Frontend Mentor](https://www.frontendmentor.io/challenges/fylo-data-storage-component-1dZPRbV5n). Frontend Mentor challenges help you improve your coding skills by building realistic projects. 
+
+## Table of contents
+
+- [Overview](#overview)
+  - [The challenge](#the-challenge)
+  - [Screenshot](#screenshot)
+  - [Links](#links)
+- [My process](#my-process)
+  - [Built with](#built-with)
+  - [Continued development](#continued-development)
+- [Author](#author)
+
+## Overview
+
+### The challenge
+
+The challenge is to build out this data storage component and get it looking as close to the design as possible.
+
+We can use any tools you like to help you complete the challenge.
+
+Your users should be able to:
```

Comment:
- Please in this task your review should also include an inline review comment, please try to add inline review comment.